doc/contributing.md

```diff
@@ -416,23 +605,74 @@ data_file:
         variant_chrom: chromosome
         variant_pos: position
         rsnum: variant_id
+        
+data_info:
+    citation: 10.1038/ng.2797
+    data_source: http://lipidgenetics.org/
+    data_type: gwas
+    context: hdl cholesterol;ldl cholesterol
+    build: b38
+    sample_size: 187167
+    n_cases: na
+    can_be_public: true
+
+method:
+    name: linear regression
+    tool: PLINK; SNPTEST; EMMAX; Merlin; GENABEL; MMAP
+    website: >-
+        http://zzz.bwh.harvard.edu/plink/download.shtml; 
+        https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html;
+        https://genome.sph.umich.edu/wiki/EMMAX;
+        https://csg.sph.umich.edu/abecasis/Merlin/tour/assoc.html;
+        http://www.genabel.org/sites/default/files/html_for_import/GenABEL_tutorial_html/GenABEL-tutorial.html;
+        https://mmap.github.io/
+
+contributor:
+    name: YoSon Park
+    github: ypar
+    email: cimrroot@gmail.com
+
+```
+
+## GWAS upload yaml file example using a Google Drive link
+
+
+For single compressed text files submissions, _cimr-d_ will accept 
+a gdrive link in place of an archive service such as 
```

Comment:
You may want to use `Goodle Drive` instead of `gdrive` (the latter seems a little too techie).